tests/test_linalg/test_approx_spectral.py

```diff
@@ -336,3 +338,11 @@ def test_negativity_approx_simple(self, psi_abc, bsz):
         actual_neg = negativity(rho_ab, DIMS[:-1], 0)
         approx_neg = negativity_subsys_approx(psi_abc, DIMS, 0, 1, bsz=bsz)
         assert_allclose(actual_neg, approx_neg, rtol=2e-1)
+
+    @pytest.mark.parametrize("bsz", [1, 2, 5])
+    def test_norm_fro_approx(self, bsz):
+        A = rand_herm(2**5)
+        actual_norm_fro = norm_fro(A)
+        approx_norm_fro = norm_fro_approx(A, tol=1e-2, bsz=bsz)
+        assert_allclose(actual_norm_fro, approx_norm_fro, rtol=1e-2)
```

Comment:
The tolerance on this assertion will need to be a lot looser so that it doesn't fail occasionally or even regularly, since the approximation is probabilistic. I'd just put in 10% for now.

Reply:
Happy to change this, but I figured it would be appropriate to expect the same level of accuracy as the `tol` passed to approx_spectral_function?

Reply:
Yes and no, you'd expect the approximation to be *on average* that good, but sometimes worse just due to statistical fluctuations. For this kinda test, it's more important that it always passes if things are roughly working rather than if the estimate is out by 1.01%.